ote_cli/ote_cli/tools/export.py

```diff
@@ -53,7 +54,7 @@ def main():
     assert args.labels is not None or args.ann_files is not None
 
     if args.labels:
-        labels = [LabelEntity(l, template.task_type) for l in args.labels]
+        labels = [LabelEntity(l, template.task_type, id=ID(f'{i}')) for i, l in enumerate(args.labels)]
```

Comment:
Not sure how mypy treats it, but simply `id=i` should work.

Reply:
fixed